### PR TITLE
{CI} Update policy config

### PR DESCRIPTION
**Pull request by wangzelin007**
**Related command**

**Description**

1. Use `\b` for more precise matching to avoid misjudgments.
2. Disable using the install keyword to match install-related issues, as the misjudgment rate is too high.

**Testing Guide**

**History Notes**

[Component Name 1] BREAKING CHANGE: `az command a`: Make some customer-facing breaking change
[Component Name 2] `az command b`: Add some customer-facing feature

---

This checklist is used to make sure that common guidelines for a pull request are followed.

- [ ] The PR title and description has followed the guideline in [Submitting Pull Requests](https://github.com/Azure/azure-cli/tree/dev/doc/authoring_command_modules#submitting-pull-requests).

- [ ] I adhere to the [Command Guidelines](https://github.com/Azure/azure-cli/blob/dev/doc/command_guidelines.md).

- [ ] I adhere to the [Error Handling Guidelines](https://github.com/Azure/azure-cli/blob/dev/doc/error_handling_guidelines.md).

### CODE REVIEW

**Comment by yonzhan** [comment]
CI

**Comment by jsntcy** [comment]
@wangzelin007, please add more details in description, for example, why we need this change? how we do it?

**Comment by wangzelin007** [comment]
Why we need to add `\b` ?
`\b` represents the word boundary in regular expressions, and more precise matching can avoid misjudgments.
Here's an example.
We have the two commands `az storage` and `az storagesync`
If we use `az storage`, will match both
If we use `\b(az storage)\b`, it will only match az storage.
image

File: .github/policies/resourceManagement.yml [hunk 1]
```diff
@@ -4477,27 +4477,27 @@ configuration:
...
-      then:
-      - addLabel:
-          label: Auto-Assign
-      - addLabel:
-          label: Installation
-      - assignTo:
-          users:
-          - jiasli
-      description: '[install] auto assign labels and users based on issue description.'
+#    - if:
```
Comment (wangzelin007) [inline .github/policies/resourceManagement.yml:4480]: The misjudgment rate of using the install keyword to match install-related issues is too high, so this task is temporarily disabled.

Comment (jsntcy) [reply .github/policies/resourceManagement.yml:4480]: So for issues really related to installation, we will not assign label automatically after this change?

Comment (wangzelin007) [reply .github/policies/resourceManagement.yml:4480]: Yes, without using keywords, we have no better way to distinguish whether it is an install-related issues.

**Review by jsntcy — APPROVED** [review]

**Pull request merged**